src/shared_utils/shared_utils.star

```diff
@@ -3,7 +3,7 @@ UDP_PROTOCOL = "UDP"
 
```

Comment:
Wait how did the docs check PR pass, given that there's no changelog update??

Reply:
i think docs-checker 0.2.0 was buggy, moved to 0.2.1 and it works (tested that it failed on no logs)